Book/causalinference.Rmd

```diff
@@ -1,12 +1,12 @@
-# Causal Inference {.tabset}
+# L'inférence causale {.tabset}
 
-Much of social science is about causality.  We might ask questions like whether [voter registration increases political participation](https://egap.org/resource/electoral-administration-in-kenya/), whether [bottom-up accountability can improve health outcomes](https://egap.org/resource/does-bottom-up-accountability-work-evidence-from-uganda/), or whether [personal narratives of immigrants help reduce prejudicial attitudes towards them](https://egap.org/resource/brief-70-how-personal-narratives-reduce-negative-attitudes-towards-immigrants-in-kenya/).
+Une grande partie des sciences sociales porte sur la causalité. On peut se demander si [l'inscription des électeurs augmente la participation politique](https://egap.org/resource/electoral-administration-in-kenya/), si [la responsabilisation bottom-up peut améliorer les perspectives en matière de santé](https://egap.org/resource/does-bottom-up-accountability-work-evidence-from-uganda/), ou si [les récits personnels des migrants aident à réduire les attitudes préjudiciables à leur égard](https://egap.org/resource/brief-70-how-personal-narratives-reduce-negative-attitudes-towards-immigrants-in-kenya/).
 
-Over the past decade, social science has become much more serious about how causal claims are made, building on a long history of work on causality dating back to the classic writings of [Fisher and Rubin](#causalinference-classics). We make greater use of experiments, and randomization has become the gold standard for addressing causal questions. 
+Au cours de la dernière décennie, les sciences sociales sont devenues beaucoup plus strictes quant à la formulation des TODO affirmations causales, en s'appuyant sur une longue histoire de travaux sur la causalité remontant aux classiques de [Fisher and Rubin](#causalinference-classics). On recourt davantage aux expériences; la randomisation est devenue la norme de référence pour répondre aux questions causales.
 
-In this module, we introduce the counterfactual approach to causal inference and how statements with causal claims can be interpreted. We introduce the potential outcome framework and how random assignment helps us make claims about what would have happened in the absence of the policy, action or program we study. We discuss the three core assumptions for causal inference: random assignment of subjects to treatment, non-interference, and excludability. 
+Dans ce module, nous introduisons l'approche contrefactuelle de l'inférence causale et comment des affirmations basées sur des TODO allégations causales peuvent être interprétées. TODO We introduce the potential outcome framework and how random assignment helps us make claims about what would have happened in the absence of the policy, action or program we study. Nous discutons des trois hypothèses de base pour l'inférence causale : TODO l'assignation aléatoire des sujets au traitement, la non-interférence et l'exclusion.
```

Comment:
non interference, assignment, + excludability --> need to precise / look up the correct words according to RCT theory